data/aboutContent.js

```diff
@@ -0,0 +1,26 @@
+const data = [
+  {
+    id: 1,
+    title: "About us",
+    paragraphy1:
+      "Blockchain will do for cooperation what the internet did for communication - We believe that blockchain technology has the potential to shift financial models as we know them, creating revenue streams that are more sustainable and fair for users. Weenvision a future in which financial infrastructures are rather decentralised and horizontal, in which any member can have a say. Through its immutability, blockchain technology providestrustworthy infrastructures that allow cooperation on a large scale.",
+    paragraphy2:
+      "Work Meaningful - With our work, we aim to impact society positively. The projects we get involved with the aim to solve significant problems in our society, sustainably and in the long run, rather than looking for a quick profit through speculation or encouraging harmful behavior.",
+  },
+
+  {
+    id: 2,
+    title: "What we do",
+    paragraphy1:
+      "Blockchain will do for cooperation what the internet did for communication - We believe that blockchain technology has the potential to shift financial models as we know them, creating revenue streams that are more sustainable and fair for users. Weenvision a future in which financial infrastructures are rather decentralised and horizontal, in which any member can have a say. Through its immutability, blockchain technology providestrustworthy infrastructures that allow cooperation on a large scale.",
+    paragraphy2:
+      "Work Meaningful - With our work, we aim to impact society positively. The projects we get involved with the aim to solve significant problems in our society, sustainably and in the long run, rather than looking for a quick profit through speculation or encouraging harmful behavior.",
+  },
+  {
+    id: 3,
+    title: "Employment",
+    paragraphy1: null,
+    paragraphy2: "We are always looking for new talent. Find out ",
+  },
+];
+export default data;
```

Comment:
have the same format `export default data` as other files `export defaut function ...`